src/renderer/pages/project/components/chat/bot_message.vue

```
@@ -4,17 +4,42 @@
+import { marked } from "marked";
+import DOMPurify from "dompurify";
+import { computed } from "vue";
+
+const props = defineProps<{
   message: string;
   time: string;
 }>();
+
+function markdownCodeBlocks(input: string): string {
+  return input.replace(/```markdown\s*\n([\s\S]*?)\n```/g, (_, code) => {
+    return marked.parse( code.trim())
```

ESLint

[error] Replace `·code.trim())` with `code.trim());`

(prettier/prettier)